src/Infrastructure/Form/Regulation/NumberedRoadFormType.php

```diff
@@ -116,7 +116,7 @@ private function getAdministratorOptions(array $administrators): array
 
         return [
             'label' => 'regulation.location.administrator',
-            'help' => 'regulation.location.administrator.help',
+            'help' => 'regulation.location.administrator.help.' . $roadType,
```

Comment:
```suggestion
            'help' => sprintf('regulation.location.administrator.help.%s', $roadType),
```

Pour une meilleure lisibilité je trouve.